pending-documentation.md

```diff
@@ -1179,6 +1179,10 @@ on a particular program by passing the `--debugger` flag to krun, or by
 invoking the llvm backend interpreter directly. Below we provide a simple
 tutorial to explain some of the basic commands supported by the LLVM backend.
 
+Before that make sure to build K with the option `-Dproject.build.type=Debug`.
+Or modify `<kdir>/llvm-backend/pom.xml` and make sure `<project.build.type>` 
+is set to `Debug`.
```

Comment:
This shouldn't be needed once the default is `RelWithDebInfo`

Reply:
That PR was closed without being merged.